src/main/scala/com/fulcrumgenomics/rnaseq/EstimateRnaSeqInsertSize.scala

```diff
@@ -1,7 +1,7 @@
 /*
  * The MIT License
  *
- * Copyright (c) 2017 Fulcrum Genomics
+ * Copyright (c) $year Fulcrum Genomics
```

Comment:
Did IntelliJ do this to you?  It should be kept as the year the file was first created.